Resources/Prototypes/Entities/Objects/Fun/crayons.yml

```diff
@@ -290,4 +290,4 @@
         whitelist:
           tags:
           - CrayonWhite
-  - type: Appearance
+  - type: Appearance
```

Comment:
```suggestion
  - type: Appearance

```

Reply:
How does this even keep happening? Is it a wrong .editorconfig setting?

Reply:
i suppose so